Dockerfile

```
@@ -0,0 +1,36 @@
+
+RUN npm run build
+
+# Production Stage
+FROM node:20-alpine
+
+WORKDIR /usr/src/app
+
+ENV NODE_ENV=production
+
+COPY package*.json ./
+RUN apk add --no-cache git
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)